lib/Site.js

```diff
@@ -338,7 +341,7 @@ Site.prototype.collectBaseUrl = function () {
     pre[path.dirname(now)] = true;
     return pre;
   }, {});
-
+  logger.verbose('Base url collected.');
```

Comment:
`baseUrl collected` (calling it "baseUrl", which is used by the user, instead of "Base url", will give user some familiarity).